b2b-sample/apps/business-admin-app/pages/o/[id].tsx

```diff
@@ -17,16 +17,17 @@
  */
 
 import { orgSignin, redirect } from "@b2bsample/shared/util/util-authorization-config-util";
+import { Session } from "next-auth";
 import { getSession } from "next-auth/react";
-import React, { useEffect } from "react";
+import { useEffect } from "react";
 import Home from "../../components/sections/home";
 
 export async function getServerSideProps(context) {
 
     const routerQuery = context.query.id;
     const session = await getSession(context);
 
-    if (session === null || session === undefined) {
+    if (session === null || session === undefined|| session.error) {
```

Comment:
`session === null || session === undefined|| session.error` -> `session === null || session === undefined || session.error`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c4229e64c82f57a16a579a24259eafab0e6cc56f